website/docs/atmo/runnable-api/logging.md

```diff
@@ -56,96 +69,97 @@ The following namespace methods are available:
 
 Logs the message with the 'info' level:
 
-<Tabs groupId="reactr-language">
-
-<TabItem value="rust" label="Rust">
+<MultiLanguageCodeBlock>
 
 ```rust
 pub fn info(msg: &str)
 ```
 
-</TabItem>
-
-<TabItem value="assemblyscript" label="AssemblyScript 🧪">
-
 ```typescript
-function logInfo(msg: string): void
+log.info(message: string): void
 ```
 
-</TabItem>
-
-<TabItem value="swift" label="Swift 🧪">
+```assemblyscript
+function logInfo(msg: string): void
+```
 
 ```swift
 public func LogInfo(msg: String)
 ```
 
-</TabItem>
-
-</Tabs>
+</MultiLanguageCodeBlock>
 
 
 ## Warn
 
 Logs the message with the 'warn' level:
 
-<Tabs groupId="reactr-language">
-
-<TabItem value="rust" label="Rust">
+<MultiLanguageCodeBlock>
 
 ```rust
 pub fn warn(msg: &str)
 ```
 
-</TabItem>
-
-<TabItem value="assemblyscript" label="AssemblyScript 🧪">
-
 ```typescript
-function logWarn(msg: string): void
+log.warn(message: string): void
 ```
 
-</TabItem>
-
-<TabItem value="swift" label="Swift 🧪">
+```assemblyscript
```

Comment:
Ah! We do have `assemblyscript` blocks

Reply:
Yes! Before this PR, AssemblyScript code was embedded in "```typescript" code blocks primarily for syntax highlighting, but also because AS was advertised as "more-or-less TypeScript".

Now we have full JS/TS support handled by Javy, and AssemblyScript can be highlighted correctly thanks to "magic" in `<MultiLanguageCodeBlock>`, so from now on the preferred usage is "```assemblyscript" code blocks.